src/Image/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+interface ImageProps {
+    alt: string;
+    globalSettings: any; // TODO: Get rid of any
+    height: string;
```

Comment:
```suggestion
    height: number;
```

Reply:
What happens if they want to have "100%" as the height? Although that's only valid for HTML 4.01 which we're not using.